src/meta.ts

```diff
@@ -0,0 +1,35 @@
+import { reactive, toRefs } from '@vue/composition-api'
+import type { MetaInfo } from 'vue-meta'
```

Comment:
Whoops - this slipped through the net.

```suggestion
import { MetaInfo } from 'vue-meta'
```